packages/vite/src/node/plugins/reporter.ts

```diff
@@ -26,6 +26,14 @@ export function buildReporterPlugin(config: ResolvedConfig): Plugin {
   const compress = promisify(gzip)
   const chunkLimit = config.build.chunkSizeWarningLimit
 
+  const numberFormatter = new Intl.NumberFormat('en', {
+    maximumFractionDigits: 2,
+    minimumFractionDigits: 2,
+  })
+  const displaySize = (bytes: number) => {
+    return `${numberFormatter.format(bytes / 1000)} kB`
```

Comment:
FYI `Intl.NumberFormat` comes with a `byte` unit that you can pass in and it will automatically format it to the appropriate unit: https://v8.dev/features/intl-numberformat#units

```ts
const numberFormatter = Intl.NumberFormat('en', {
	notation: 'compact',
	style: 'unit',
	unit: 'byte',
	unitDisplay: 'narrow',
	minimumFractionDigits: 1,
	maximumFractionDigits: 2,
}).format;
```

Reply:
Nice! PR welcome to improve this